.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+on:
+  push:
+    branches: [master]
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version: ['1.0', '1']  # Test against LTS and current minor release
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+        arch: [x64]
+        include:
+          # Also test against Windows 32-bit on LTS.
+          - version: '1.0'
```

Comment:
Is it worth testing against windows 32bit considering it is the second tier in Julia support?

Reply:
This is what we were doing previously, so I kept it. This is where having system information of who is installing each package would be useful...

Reply:
32-bit linux would probably be more useful to test than 32-bit windows, but they provide about the same coverage of the code